internal/docker/windowsservercore-2019

```diff
@@ -16,10 +16,14 @@ FROM cirrusci/windowsservercore:visualstudio2022-2022.06.23
 SHELL ["powershell.exe", "-NoLogo", "-NoProfile", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 RUN choco install -y rustup.install; \
-    Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1; \
-    refreshenv; \
-    rustup toolchain add beta; \
-    rustup component add rustfmt clippy; \
-    Remove-Item C:\ProgramData\chocolatey\logs\* -Force -Recurse ; \
-    Remove-Item $Env:localappdata\Nuget\Cache\*nupkg ; \
-    Remove-Item $Env:temp\* -Force -Recurse
+Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1; \
+refreshenv; \
+rustup toolchain add beta; \
+rustup component add rustfmt clippy; \
+Remove-Item C:\ProgramData\chocolatey\logs\* -Force -Recurse ; \
+Remove-Item $Env:localappdata\Nuget\Cache\*nupkg ; \
+Remove-Item $Env:temp\* -Force -Recurse ; \
+mkdir "C:\Temp\elkodon\services" ; \
+mkdir "C:\Temp\elkodon\tests" ; \
+mkdir "C:\Temp\elkodon\shm" ; \
+icacls "C:\Temp" /t /c /grant Everyone:F
```

Comment:
Please add the indentation back. It makes it easier to distinguish from docker build commands and actual windows commands